util/admin/admin.go

```diff
@@ -316,6 +316,9 @@ const (
 func CheckIndicesCount(ctx sessionctx.Context, dbName, tableName string, indices []string) (byte, int, error) {
 	// Here we need check all indexes, includes invisible index
 	ctx.GetSessionVars().OptimizerUseInvisibleIndexes = true
+	defer func() {
+		ctx.GetSessionVars().OptimizerUseInvisibleIndexes = false
```

Comment:
Set it to the original value is better？

Reply:
It's ok to set it to false here. Because we use the pattern:
``` SQL
set it to true
lalala
reset it
```

Reply:
Got it.